Sources/SwiftGodot/Extensions/ClassInfo+ConvenienceProperties.swift

```diff
@@ -0,0 +1,263 @@
+//
+//  ClassInfo+ConvenienceProperties.swift
+//  
+//
+//  Created by Marquis Kurt on 5/29/23.
+//
+
+import Foundation
+import UniformTypeIdentifiers
+
+public protocol Nameable {
+    var name: String { get }
+}
+
+@available(macOS 11.0, *)
+extension UTType {
+    /// The file type that corresponds to a Godot resource file.
+    public static var godotResource = UTType(filenameExtension: "res")
+
+    /// The file type that corresponds to a Godot scene file.
+    public static var godotScene = UTType(filenameExtension: "tscn")
+
+    /// The file type that corresponds to a GDScript file.
+    public static var gdscript = UTType(filenameExtension: "gd")
```

Comment:
Capital "s" in Script?

Reply:
I considered this, but it didn't feel right to me as GDScript is one word rather than "GD Script".